Automattic/wp-calypso
# Update app banners design

#### Changes proposed in this Pull Request

This PR updates the design of the mobile app banners:

| | Editor | Notifications | Reader | Stats |
| --- | --- | --- | --- | --- |
| **Before** | ![editor-old](https://user-images.githubusercontent.com/830056/118190061-f8b8af80-b418-11eb-91db-719e1735a694.png) | ![notifications-old](https://user-images.githubusercontent.com/830056/118190066-fa827300-b418-11eb-8b02-1ac25b974cb4.png) | ![reader-old](https://user-images.githubusercontent.com/830056/118190072-fbb3a000-b418-11eb-822f-c1bda443d07a.png) | ![stats-old](https://user-images.githubusercontent.com/830056/118190076-fc4c3680-b418-11eb-966c-37c5bdd2c9ba.png) |
| **After** | ![editor-new](https://user-images.githubusercontent.com/830056/118190058-f7878280-b418-11eb-9249-e9ce3e4b72ab.png) | ![notifications-new](https://user-images.githubusercontent.com/830056/118190064-f9e9dc80-b418-11eb-8524-508f1e331f67.png) | ![reader-new](https://user-images.githubusercontent.com/830056/118190068-fb1b0980-b418-11eb-91b1-8abf616ce86d.png) | ![stats-new](https://user-images.githubusercontent.com/830056/118190074-fc4c3680-b418-11eb-898c-73a706d6e111.png) |

For additional context behind this change, please check this internal reference: pbArwn-2bY-p2

**Note:** I'm not a web developer, so please let me know if something is wrong or can be improved.

The changes in this PR are 100% visual and all existing behavior should remain the same. 

Most of those changes were done in the stylesheet of the App Banner component, including changes to the colors, sizes, paddings and margins of nearly all of its elements. I made sure to keep commits separated for each of those elements, so it should be easy to follow the changes when reviewing them.

The changes that were made outside the stylesheet were due to the removal of all illustrations that were previously being used as the card background.

I tried to follow all the available guidelines when making those changes, but let me know if I'm missing something of if there's a better way of achieving it.

#### Testing instructions

The app banner is only visible on mobile browsers, so one way to test it is to use [the Device Mode feature of DevTools to simulate a mobile device](https://developer.chrome.com/docs/devtools/device-mode/). Once you've done that, the banners should show up for any site that has been launched and is older than 1 hour (unless you've dismissed it before).

1. Navigate to the following sections: Editor, Notifications, Reader and Stats.
2. Verify that the app banner shows up correctly and that its design is updated.
3. Verify that all existing behavior is the same as it was before.
